rs/src/contracts/tunnel_access_control_entry.rs

```diff
@@ -26,8 +26,9 @@ pub struct TunnelAccessControlEntry {
     // the user/group/org IDs. It may be one of the well-known provider names in
     // `TunnelAccessControlEntry.Providers`, or (in the future) a custom identity
     // provider.  For public key ACEs, this value is the type of public key, e.g. "ssh".
-    // For IP address range ACEs, this value is the IP addrss version, e.g. "ipv4" or
-    // "ipv6".  For anonymous ACEs, this value is null.
+    // For IP address range ACEs, this value is the IP address version, "ipv4" or "ipv6",
```

Comment:
Looks like this was missed in the related PR - does the rust contract generator not run automatically?

Reply:
It runs during the C# SDK build. It should have been updated since there were C# changes in that PR.